### ISSUE TITLE
If...else if... braces

### ISSUE THREAD

**Issue by RealyUniqueName** [adireddy/haxe-checkstyle#38]
Using `else if` construction:

```
if (condition) {
    someAction();
} else if (condition2) {
    anotherAction();
}
```

produces an issue:

```
Warning: LINE - 264: No braces used for body of else branch
```

Related part of checkstyle config:

```
       {
            "type"  : "NeedBraces",
            "props" : {
                "severity"                 : "WARNING",
                "allowSingleLineStatement" : true,
                "tokens"                   : ["IF", "ELSE_IF", "FOR", "WHILE", "TRY", "CATCH"]
            }
        },
```

Is there a way to allow `else if` without surrounding `if` with curly braces?

**Comment by AlexHaxe** [comment]
I have updated `token-tree`branch to address `else if` cases. 
As long as `else` and `if` + condition are on the same line, it should not trigger a message.

However a wrapped condition will trigger a message:

```
if (condition) {
    someAction();
} else if (condition2 &&
    condition3) {
    anotherAction();
}
```

**Comment by RealyUniqueName** [comment]
Thanks!
